routes/index.js

```diff
@@ -4,9 +4,13 @@ import express from "express";
 import userRoute from "./user/index.js";
 import attendenceRoute from "./attendence/index.js";
 import miscRoute from "./misc/index.js";
+
```

Comment:
Remove these empty lines...

Reply:
There should not be any empty line in import section. and all imports should be on top.